mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
The code `self._message.get('to', self._message.get('delivered-to'))` already returns `None `if `to `or `delivered-to` are `None`.
This is the [documentation](https://docs.python.org/2/library/email.message.html#email.message.Message.get)

So I can't understand:

```
 def to_(self):
        to_ = self._message.get('to')        
        if not to_:
             return None
        else:
             return decode_header_part(
                 self._message.get('to', self._message.get('delivered-to')))
```